components/liquidity/MarketLiquiditySection.tsx

```diff
@@ -119,7 +121,7 @@ const LiquidityHeader = ({ market }: { market: FullMarketFragment }) => {
         </LiquidityHeaderButtonItem>
       </div>
       <LiquidityModal
-        poolId={pool.poolId}
+        poolId={pool!.poolId}
```

Comment:
Maybe this should be fixed on the sdk too

Reply:
It's ok if pool doesn't exist, but think sdk should remove `null` usage on most types if possible

Reply:
Tried adding new type to use in components in this file
```
type FullMarketFragmentWithPool = FullMarketFragment & {
  pool: NonNullable<FullMarketFragment["pool"]>;
};
```

but then code on marketId page complains. Should I add the type above and the type guard for it or using `pool!` is fine in this case since pool is required for those components to work